incubator/virtualcluster/pkg/syncer/mccontroller/mccontroller.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
+	priorityclassv1 "k8s.io/api/scheduling/v1"
```

Comment:
Do you mind change this one to schedulingv1 as well?

Reply:
yes